server/src/modules/Auth/auth.controller.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  Controller,
+  Get,
+  Request,
+  Response,
+  Post,
+  HttpStatus,
+  Body } from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { LocalPayload } from './interfaces/local-payload.interface';
+
+@Controller('auth')
+export class AuthController {
+  constructor(private readonly authService: AuthService) {}
+
+  @Post('login')
+  async loginUser(@Response() res, @Body() body: LocalPayload, @Request() req) {
+    if (!(body && body.email)) {
+      return res.status(HttpStatus.FORBIDDEN).json({ message: 'Email are required!' });
+    }
+    try {
+      const user = await this.authService.validateUser(body);
+      req.session.user = user;
```

Comment:
As I understand, you serialize into db entire user (I haven't find, where you do serialization of selected fields). If so, It is not good approach, because user can change during work of application so we could see old information and make hacks to invalidate this "cache". Session, in common approach, stores only user identifier, and make extra request for data into db.

Reply:
done, just id has stored